invenio_rdm_records/services/schemas/parent/__init__.py

```diff
@@ -46,6 +46,7 @@ class RDMParentSchema(ParentSchema, FieldPermissionsMixin):
         values=fields.Nested(PIDSchema),
         dump_only=True,
     )
+    is_verified = fields.Boolean(dump_only=True)
```

Comment:
question: do we want to add permissions to read this file?

e.g. in the users service we added:


```python
can_read_system_details = [AdminAction(user_management_action)]
```

Reply:
I don't see it has a sensitive field. What is the behaviour in Zenodo today?

Reply:
Zenodo does not dump the field, therefore is not visible in API queries.

Ideally, we could protect the field and only make it visible to admins. However, it would "pollute" a bit of the record's namespace by introducing user-related permission.

Options:

1 - Do not dump the field at all (I don't see why not)
2 - Add a permission to dump it (e.g. `AdminAction(user_management_action)`)

WDYT?

Reply:
I would personally leave it like this. Can you write this down to discuss it with Alex/Lars?

Reply:
Created an issue https://github.com/inveniosoftware/invenio-rdm-records/issues/1411

And added it to the epic Tasks https://github.com/inveniosoftware/invenio-app-rdm/issues/2290